packages/styles/scss/components/footer/_footer.scss

```diff
@@ -41,7 +41,7 @@
       }
     }
 
-    .#{$prefix}--link {
+    li a.#{$prefix}--link {
```

Comment:
@RaphaelAmadeu wonder if we should add some specificity rather than target the element:
```
.#{$prefix}--footer.#{$prefix}--link {
```

Reply:
@jeffchew 
Just updated the PR with some changes regarding this,
Thanks!